src/UglyToad.PdfPig/Content/IPdfImage.cs

```diff
@@ -27,17 +27,6 @@ public interface IPdfImage
         /// </summary>
         int HeightInSamples { get; }
 
-        /// <summary>
-        /// The <see cref="ColorSpace"/> used to interpret the image.
-        /// This defines the number of color components per sample, e.g.
-        /// 1 component for <see cref="Graphics.Colors.ColorSpace.DeviceGray"/>,
-        /// 3 components for <see cref="Graphics.Colors.ColorSpace.DeviceRGB"/>,
-        /// 4 components for <see cref="Graphics.Colors.ColorSpace.DeviceCMYK"/>,
-        /// etc.
-        /// This is not defined where <see cref="IsImageMask"/> is <see langword="true"/> and is optional where the image is JPXEncoded for <see cref="XObjectImage"/>.
-        /// </summary>
-        ColorSpace? ColorSpace { get; }
```

Comment:
We'll need to document a migration path for these public API members, if you're able to pop a brief summary in a comment that would be great thanks

Reply:
@EliotJones makes sense, I'll post something here